packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -11,49 +11,55 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
-
-:root {
-  --spectrum-searchwithin-width: 250px;
+.spectrum-SearchWithin {
+  --spectrum-searchwithin-width: 240px;
+  --spectrum-searchwithin-searchfield-width: 128px;
 
   /* Force override */
   --spectrum-searchwithin-border-radius: 0;
-  --spectrum-searchwithin-dropdown-min-width: 0;
+  --spectrum-searchwithin-picker-min-width: 0;
 }
 
 .spectrum-SearchWithin {
-  width: var(--spectrum-searchwithin-width);
+  inline-size: var(--spectrum-searchwithin-width);
   display: inline-flex;
   position: relative;
+}
 
-  .spectrum-Dropdown {
-    width: auto;
-    min-width: 0;
-  }
+.spectrum-SearchWithin-picker {
+  inline-size: auto;
+  min-inline-size: 0;
+  flex-shrink: 0;
 
-  .spectrum-Dropdown-trigger {
-    border-top-right-radius: var(--spectrum-searchwithin-border-radius);
-    border-bottom-right-radius: var(--spectrum-searchwithin-border-radius);
+  > button {
+    contain: unset;
+    border-end-start-radius: var(--spectrum-searchwithin-border-radius);
+    border-start-start-radius: var(--spectrum-searchwithin-border-radius);
   }
+}
+
+.spectrum-SearchWithin-searchfield {
+  width: var(--spectrum-searchwithin-searchfield-width);
```

Comment:
`min-inline-size`

Reply:
this should fix this case 
<img width="141" alt="Screen Shot 2021-07-27 at 4 24 54 PM" src="https://user-images.githubusercontent.com/698229/127240115-0f27511d-835f-4a6b-90c8-0508110008e5.png">